int32 total_timesteps
int32 episode_number
int32 episode_timesteps
float32 episode_reward
